src/mlpack/tests/main_tests/kde_test.cpp

```diff
@@ -88,7 +88,7 @@ BOOST_AUTO_TEST_CASE(KDEGaussianRTreeResultsMain)
 
   // Check whether results are equal.
   for (size_t i = 0; i < query.n_cols; ++i)
-    BOOST_REQUIRE_CLOSE(kdeEstimations[i], mainEstimations[i], relError);
+    BOOST_REQUIRE_CLOSE(kdeEstimations[i], mainEstimations[i], 100 * relError);
```

Comment:
What's surprising about this is that it highlights how loose our bounds were before: when the tests passed with `relError`, it means that KDE was achieving relative error 100 times tighter than what was asked for.  I imagine that the speedup from the new pruning rules and bounds is very significant!